app/User.php

```diff
@@ -35,11 +35,19 @@ public function sharedFiles(){
         return $this->hasMany('App\Share', 'idUser');
     }
 
+
+     public function permission(){
+        return $this->hasOne('App\UserType', 'id', 'idUserType');
```

Comment:
Should be belongs according to Laravel standards.

 UserType may be a better name but you can leave it as permission